src/settings.rs

```diff
@@ -23,33 +24,14 @@ pub struct Settings {
     pub giganto_ingest_srv_addr: SocketAddr, // IP address & port to giganto
     #[serde(deserialize_with = "deserialize_socket_addr")]
     pub giganto_publish_srv_addr: SocketAddr, // IP address & port to giganto
-    pub review_name: String,    // host name to review
+    pub manager_name: String,   // host name to manager
     #[serde(deserialize_with = "deserialize_socket_addr")]
-    pub review_rpc_srv_addr: SocketAddr, // IP address & port to review
+    pub manager_rpc_srv_addr: SocketAddr, // IP address & port to manager
```

Comment:
We need to remove these two from the `Settings` struct. We also need to remove `cert`, `key`, and `ca_certs` from the struct. This is because we provide these info as command line arguments and options.

Reply:
The fields have been removed.